### ISSUE TITLE
Profiling numbers are wrong

### ISSUE THREAD

**Issue by sebastianelsner** [CGRU/cgru#438]
From time to time I get these numbers in the logs:

```
Thu 07 Feb 20:08.02: Server load profiling:
Clients per second: 282.16, Now: 64 (processed 409600 connections in last 1451.65 seconds).
Prep: -18367622028115080.00, Proc: 5.18, Post: 18367622028115248.00, Total: 172.00 ms
```
-18367622028115080.00 for exampe seems to be really wrong.  Where can this come from?

**Comment by timurhai** [comment]
Hm. It seems like overflow. I`ll check it.

**Comment by timurhai** [comment]
Hi.
Sorry can't reproduce or even imagine double arithmetic overflow.
May be i am looking for error in a wrong place.
It is time for 2.3.1 release due some important bug fixes.
Lets keep this issue, but delay it.

**Comment by sebastianelsner** [comment]
Ok,  will do an upgrade soon-ish anyways. Might be a CentOS issue as well...

**Comment by timurhai** [comment]
Hi!
So, have anybody noticed invalid profiling numbers last year?

**Comment by sebastianelsner** [comment]
Yes I see them all the time, but we still have not done any upgrades...
